README.md

```diff
@@ -29,6 +29,8 @@ if (!requireNamespace("BiocManager", quietly = TRUE))
 BiocManager::install(c("DECIPHER","sangerseqR","ape", "BiocStyle"))
```

Comment:
we need to that a comment about force = TRUE, stating already here that a user might need to set it up as true if older versions of these packages are already installed

Reply:
Addressed in commit 411c4a41425e204d55aa0b1da80a88023b390b4e